demo.html

```diff
@@ -1,25 +1,45 @@
-<style>
- bit-json-editor { height: 200px; width: 600px;}
-</style>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>bit-json-editor demo</title>
+		<style>
+			bit-json-editor {
+				height: 200px;
+				width: 600px;
+			}
+		</style>
+	</head>
+	<body>
+		<my-app></my-app>
 
+		<script type="module">
+			import "./bit-json-editor.mjs";
+			import {
+				route,
+				StacheElement,
+				ObservableObject
+			} from "//unpkg.com/can@pre/core.mjs";
 
-<my-app></my-app>
+			class MyApp extends StacheElement {
+				static view = `
+					<bit-json-editor data:from="this.data" />
+				`;
+				static props = {
+					data: {
+						get default() {
+							const observable = new ObservableObject();
+							route.data = observable;
```

Comment:
I think you could depend on the default `route.data` instead of creating your own.

Reply:
done.